src/Forms/DatetimeField.php

```diff
@@ -162,11 +341,12 @@ protected function getISO8601Formatter()
      */
     public function setValue($value, $data = null)
     {
+        // Save raw value for later validation
+        $this->rawValue = $value;
+
         // Empty value
         if (empty($value)) {
             $this->value = null;
-            $this->dateField->setValue(null);
-            $this->timeField->setValue(null);
             return $this;
         }
         if (is_array($value)) {
```

Comment:
setSubmittedValue() doesn't support array format anymore. We should fail if not a string.